.github/helper/semgrep_rules/report.yml

```
@@ -0,0 +1,19 @@
+rules:
+- id: frappe-missing-translate-function-in-report-python
+  paths:
+    include:
+    - "**/report"
```

```suggestion
  paths:
    include:
    - "**/report"
    exclude:
    - "**/regional"
```

Checked current failures, more than 1/2 are regional reports. They won't need translations.